docs/apidiffs/current_vs_latest/opentelemetry-exporter-jaeger-proto.txt

```diff
@@ -0,0 +1,1355 @@
+Comparing source compatibility of  against 
++++  NEW CLASS: PUBLIC(+) FINAL(+) io.opentelemetry.exporter.jaeger.proto.api_v2.Collector  (not serializable)
+	+++  CLASS FILE FORMAT VERSION: 52.0 <- n.a.
+	+++  NEW METHOD: PUBLIC(+) STATIC(+) com.google.protobuf.Descriptors$FileDescriptor getDescriptor()
+	+++  NEW METHOD: PUBLIC(+) STATIC(+) void registerAllExtensions(com.google.protobuf.ExtensionRegistryLite)
+	+++  NEW METHOD: PUBLIC(+) STATIC(+) void registerAllExtensions(com.google.protobuf.ExtensionRegistry)
++++  NEW CLASS: PUBLIC(+) STATIC(+) FINAL(+) io.opentelemetry.exporter.jaeger.proto.api_v2.Collector$PostSpansRequest  (not serializable)
```

Comment:
I'd definitely prefer it if we don't have to expose any of this stuff as a public API for otel.

Reply:
We currently already do - so you'd like to try removing them from the public API and see if anyone has issues?

Reply:
how were they not in the apidiffs previously?

Reply:
oh, is this because we only generate diffs, and they've been public from the beginning of time? 😢

Reply:
Sort of - the huge removal in the other artifact correspondence to this addition. So the API has moved artifacts while otherwise unchanged in its publicness from before.

Reply:
yeah. ugh. This sort of makes me want to not touch this module, since this is a pretty big breaking change, if someone happens to be relying on this stuff for some unknown reason. I need to think on this a bit.

Reply:
Sure - FWIW, with an artifact move and switch to compileOnly, it's easy to add this artifact to a build if the classes are being used. We don't consider needing additional POM changes to be semconv breaking so came up with this approach to try to keep it non-breaking.